client/src/localization/languages/He.tsx

```diff
@@ -0,0 +1,380 @@
+  com_nav_hide_panel: 'הסתר לוח הצד הימני ביותר',
+  com_nav_modular_chat: 'אפשר החלפת נקודות קצה באמצע שיחה',
+  com_nav_latex_parsing: 'ניתוח LaTeX בהודעות (עשוי להשפיע על הביצועים)',
+  com_nav_profile_picture: 'תמונת פרופיל',
+  com_nav_change_picture: 'שנה תמונה',
+  com_nav_plugin_store: 'חנות פלאגין',
+  com_nav_plugin_install: 'התקן',
+  com_nav_plugin_uninstall: 'הסר התקנה',
+  com_nav_tool_add: 'הוסף',
+  com_nav_tool_remove: 'הסר',
+  com_nav_tool_dialog: 'כלי סייען',
+  com_nav_tool_dialog_description: 'יש לשמור את האסיסטנט כדי להמשיך בבחירת הכלים.',
+  com_show_agent_settings: 'הצג הגדרות סוכן',
+  com_show_completion_settings: 'הצג הגדרות השלמה',
+  com_hide_examples: 'הסתר דוגמאות',
+  com_show_examples: 'הצג דוגמאות',
+  com_nav_plugin_search: 'תוספי חיפוש',
+  com_nav_tool_search: 'כלי חיפוש',
+  com_nav_plugin_auth_error: 'אירעה שגיאה בניסיון לאמת את הפלאגין הזה. בבקשה נסה שוב.',
+  com_nav_export_filename: 'שם קובץ',
+  com_nav_export_filename_placeholder: 'הגדר את שם הקובץ',
+  com_nav_export_type: 'סוג',
+  com_nav_export_include_endpoint_options: 'כלול אפשרויות נקודת קצה',
+  com_nav_enabled: 'מופעל',
+  com_nav_not_supported: 'לא נתמך',
+  com_nav_export_all_message_branches: 'ייצא את כל ענפי ההודעות',
+  com_nav_export_recursive_or_sequential: 'רקורסיבי או רציף?',
+  com_nav_export_recursive: 'רקורסיבי',
+  com_nav_export_conversation: 'ייצא שיחה',
+  com_nav_theme: 'נושא',
+  com_nav_theme_system: 'מערכת',
+  com_nav_theme_dark: 'כהה',
+  com_nav_theme_light: 'אור',
+  com_nav_user_name_display: 'הצג שם משתמש בהודעות',
+  com_nav_clear_all_chats: 'נקה את כל השיחות',
+  com_nav_confirm_clear: 'אשר נקה',
+  com_nav_close_sidebar: 'סגור סרגל צד',
+  com_nav_open_sidebar: 'פתח סרגל צד',
+  com_nav_send_message: 'שלח הודעה',
+  com_nav_log_out: 'צא',
+  com_nav_user: 'USER',
+  com_nav_clear_conversation: 'נקה שיחות',
+  com_nav_clear_conversation_confirm_message:
+    'אתה בטוח שאתה רוצה לנקות את כל השיחות? זה בלתי הפיך.',
+  com_nav_help_faq: 'עזרה ושאלות נפוצות',
+  com_nav_settings: 'הגדרות',
+  com_nav_search_placeholder: 'חפש הודעות',
+  com_nav_setting_general: 'כללי',
+  com_nav_setting_beta: 'תכונות ביטא',
+  com_nav_setting_data: 'בקרות נתונים',
+  com_nav_setting_account: 'חשבון',
+  com_nav_language: 'שפה',
+  com_nav_lang_auto: 'זיהוי אוטומטי',
+  com_nav_lang_english: 'English',
+  com_nav_lang_chinese: '中文',
+  com_nav_lang_german: 'Deutsch',
+  com_nav_lang_spanish: 'Español',
+  com_nav_lang_french: 'Français ',
+  com_nav_lang_italian: 'Italiano',
+  com_nav_lang_polish: 'Polski',
+  com_nav_lang_brazilian_portuguese: 'Português Brasileiro',
+  com_nav_lang_russian: 'Русский',
+  com_nav_lang_japanese: '日本語',
+  com_nav_lang_swedish: 'Svenska',
+  com_nav_lang_korean: '한국어',
+  com_nav_lang_vietnamese: 'Tiếng Việt',
+  com_nav_lang_traditionalchinese: '繁體中文',
+  com_nav_lang_arabic: 'العربية',
+  com_nav_lang_turkish: 'Türkçe',
+  com_nav_lang_dutch: 'Nederlands',
+  com_nav_lang_indonesia: 'Indonesia',
```

Comment:
remove this

Reply:
Don't you think 
  com_nav_language: 'שפה',
 Is still needed? 
 I know on other systems that when the setup is in the language you don't know, understanding the language menu is crucial...

Reply:
exact, that's why language should be only in English. If someone has some strange bugs and doesn't understand the language that Librechat is set to, he approximately needs to know where to change it. The setting button has an icon, but the language label doesn't have it

Reply:
You are right
Will fix it later